src/main/java/com/org/checkr/controller/CandidatesController.java

```diff
@@ -0,0 +1,34 @@
+package com.org.checkr.controller;
+
+import com.org.checkr.dto.CandidateDTO;
+import com.org.checkr.entity.Candidate;
+import com.org.checkr.service.CandidateService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/candidates")
+public class CandidatesController {
+    @Autowired
```

Comment:
can u check if instrad of @autowire use constructor injection for better standard